integration_test/agents/agents.go

```diff
@@ -638,10 +630,6 @@ func RunInstallFuncWithRetry(ctx context.Context, logger *log.Logger, vm *gce.VM
 	shouldRetry := func(err error) bool { return isRetriableInstallError(vm.Platform, err) }
 	installWithRecovery := func() error {
 		err := installFunc()
-		if err != nil && shouldRetry(err) && vm.Platform == "rhel-8-1-sap-ha" {
-			logger.Println("attempting recovery steps from https://access.redhat.com/discussions/4656371 so that subsequent attempts are more likely to succeed... see b/189950957")
-			gce.RunRemotely(ctx, logger, vm, "", "sudo dnf clean all && sudo rm -r /var/cache/dnf && sudo dnf upgrade")
-		}
```

Comment:
Ditto…

Reply:
Added it back for `rhel-8-*-sap-ha`.

Reply:
Just curious, is this code not needed anymore?

Reply:
Oops, I misplaced your comment in my head -- will fix this too